pkg/skaffold/config/config.go

```diff
@@ -35,15 +35,16 @@ type SkaffoldConfig struct {
 
 // BuildConfig contains all the configuration for the build steps
 type BuildConfig struct {
-	Artifacts []*Artifact `yaml:"artifacts,omitempty"`
-	TagPolicy TagPolicy   `yaml:"tagPolicy,omitempty"`
+	Artifacts []*Artifact `yaml:"artifacts"`
```

Comment:
Please don't remove the `omitempty` on those two lines. It's useful

Reply:
Ah sorry, merge conflict issue.